### ISSUE TITLE
Laravel 5 (v.3.4.0) manyToMany sync() doesn't update position

### ISSUE THREAD

**Issue by IvanRibakov** [boxfrommars/rutorika-sortable#33]
I'm using Laravel 5 and have updated plugin to version 3.4.0. Everything seems to work as expected during the seeding - records are created using `attach()` method and `position` column on the pivot table is populated correctly. However, during the edit, when sending to controller reordered list of child records and using `sync()` to update parent record, no changes to `position` column take place.

Any suggestions?

**Comment by boxfrommars** [comment]
@IvanRibakov Hm, all tests are passed. Can you create testcase for this bug?

**Comment by boxfrommars** [comment]
or just show your request data and controller

**Comment by IvanRibakov** [comment]
Relevant part of the request data where `67` and `18` are color IDs that are already attached and have position values `18 => 1` and `67 => 2`:
```
array:12 [
  "_method" => "PATCH"
  ...
  "colors" => array:2 [
    0 => "67"
    1 => "18"
  ]
]
```

And controller method:
```
public function update(SchoolRequest $request, School $school)
{
    $school->update($request->all());
    $school->colors()->sync($request->colors);
    return redirect()->route('backend.schools.show', ['school' => $school->id])
        ->with('success', 'School updated successfully.');
}
```

**Comment by boxfrommars** [comment]
Thank you. I think I found what is going wrong. I'll try to fix it today.

**Comment by boxfrommars** [comment]
However you can fix it in your controller by detaching all colors before syncing:
```
<?php
public function update(SchoolRequest $request, School $school)
{
    $school->update($request->all());
    $school->colors()->detach(); # <-- this line
    $school->colors()->sync($request->colors);
    return redirect()->route('backend.schools.show', ['school' => $school->id])
        ->with('success', 'School updated successfully.');
}
```

**Comment by boxfrommars** [comment]
@IvanRibakov Fixed in 3.4.1
